R/lazyframe__lazy.R

```diff
@@ -1,56 +1,56 @@
-#' @title Inner workings of the LazyFrame-class
+#' @title Inner workings of the PolarsLazyFrame-class
 #'
-#' @name LazyFrame_class
-#' @description The `LazyFrame`-class is simply two environments of respectively
+#' @name PolarsLazyFrame_class
```

Comment:
if you change the "name" arg here, then it will change the name of the Rd files and therefore it will also break the "docs" workflow. So either these shouldn't be changed or the script "make-docs.R" should be updated